packages/react-import-remote/README.md

```diff
@@ -66,17 +66,13 @@ Source of the script to load the global from
 
 Generates a preconnect link tag for the source’s domain using [`@shopify/react-preconnect`](https://github.com/Shopify/quilt/tree/master/packages/react-preconnect)
 
-**onError**
-
-Callback that takes in `error` is called if an error occurs
-
 **getImport**
 
 Callback that takes in `window` with the added global and returns the global added to the `window` by the new script
 
 **onImported**
 
-Callback that gets called with the imported global
+Callback that gets called with the imported global or an `error` if one occurs
```

Comment:
How do you feel about this change? I am still kind of waffling back and forth.

Reply:
I think it's fine. I always felt like there were too many props on this component. 

As long as we have an example, which I added in the Readme.